test/perf/micro/perf.f90

```diff
@@ -190,6 +190,15 @@ integer function parse_int(s, base) result(n)
 
 end function
 
+subroutine printfd(n)
+integer, intent(in) :: n
+integer :: i , unit
+open(unit=1, file="foo")
+do i = 1, n
+  write(unit=1, fmt=*) i, i
+end do
+close(unit=1)
```

Comment:
So this subroutine should be written as follows:
```fortran
subroutine printfd(n)
integer, intent(in) :: n
integer :: i, u
open(newunit=u, file="foo", status="replace")
do i = 1, n
  write(u, *) i, i
end do
close(u)
end subroutine
```

Reply:
@certik: thanks, I'll merge those fixes in.